### ISSUE TITLE
Collect functional requirements

### ISSUE THREAD

**Issue by hkakutalua** [gdgluanda/rapidinho#1]
Collect functional requirements for the application by talking with the customer.
Functional requirements can be brainstormed inside the team or tweaked if needed.
Top level priority "should" be given to immediate functionalities for the first version of the app.

**Comment by hkakutalua** [comment]
@pedromassango, could you please talk with any Rapidinho employee to collect relevant functionalities (and of course take note of all them)? Before the weekend would be perfect.

You can also pass his/her phone contact to us, so we can (re)collect as well to be sure, right now it's impossible to create a reunion with the presence of all of us.

**Comment by pedromassango** [comment]
I will talk with someone in saturday.

**Comment by pedromassango** [comment]
I gues that we should not use their name as the name of the project, but the name of the app like Delivery App

**Comment by hkakutalua** [comment]
Yes, i agree with your last point, for one reason particulary: this is a open-source project with a very permissive license, i do agree that we should keep the license as is, so that people know that GDG Luanda organizers made the app.

But, i recommend we change the name of the app to a generic one, in the future people will fork/copy the project code, so it's not cool to leave Rapidinho name in the code.

My plan it's just to create the app for our sake and free-time, and then show them to evaluate and maybe sell. If we reach that phase of selling them the app, we will of course create a private fork from this project. That's just my vision, what do you think?

Of course, creating a fork and maintaining it up to date with the original code will be a pain, so in that case the second option would be maintain all our work open-source with the app name as rapidinho to simplify things.

**Comment by braulio94** [comment]
Let's talk to Rapidinho team (though we kinda have already) and see what they say. 
If they're ok with us using their logo and name we keep developing otherwise we change it to a generic name as @Henry-Keys suggested.

**Comment by hkakutalua** [comment]
@braulio94, we can do that otherwise.

**Comment by pedromassango** [comment]
I talked with one of Rapidinho's employee, unfortunately she doesn't have any information about the app requeriments. She will talk with her boss to talk to speak with us next saturday.

**Comment by braulio94** [comment]
Thanks @pedromassango. 

The idea was not to ask them for the app requirements, it's simply to let them know that we're building this app and we're using their brand name and logo.

Basically, we just want their consent.
